easypost/services/api_key_service.py

```diff
@@ -0,0 +1,45 @@
+from typing import (
+    Any,
+    Dict,
+    List,
+)
+
+from easypost.easypost_object import convert_to_easypost_object
+from easypost.models import ApiKey
+from easypost.requestor import (
+    RequestMethod,
+    Requestor,
+)
+from easypost.services.base_service import BaseService
+
+
+class ApiKeyService(BaseService):
+    def __init__(self, client):
+        self._client = client
+        self._model_class = ApiKey.__name__
+
+    def all(self) -> Dict[str, Any]:
+        """Retrieve a list of all API keys."""
+        url = "/api_keys"
+
+        response = Requestor(self._client).request(method=RequestMethod.GET, url=url)
+
+        return convert_to_easypost_object(response=response)
+
+    def retrieve_api_keys_for(self, id: str) -> List[ApiKey]:
```

Comment:
We should name this function as `api_keys()` to match other CLs

Reply:
Resolved per our internal conversations.